docs/cloud/cloud-query/cloud-query-execute.md

```diff
@@ -39,10 +36,20 @@ Write a query using either the standard [SQL Language](/docs/sql-guide/sql-guide
 
 ### Example
```

Comment:
Nope. Don't stick examples in the middle of procedures.

This is because the user will be following the procedure, then be stopped from progressing because of something else.

This is why I've got examples at the bottom of the SQL guide pages.